include/seqan3/alphabet/nucleotide/rna5.hpp

```diff
@@ -70,6 +70,9 @@ class rna5 : public nucleotide_base<rna5, 5>
 
     //!\brief Allow implicit construction from dna/rna of the same size.
     constexpr rna5(dna5 const & r) noexcept
+#if SEQAN3_WORKAROUND_GCC_90897
+        requires true
+#endif
```

Comment:
Yes, I hoped the other ones were not triggered by code, but apparently they are.